src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -103,10 +105,20 @@ export default class MxGraphRenderer {
             mxEdge.geometry.offset = new mxPoint(relativeLabelX, relativeLabelY);
           }
         }
+
+        this.insertMessage(edge, mxEdge);
       }
     });
   }
 
+  private insertMessage(edge: Edge, mxEdge: mxCell): void {
+    if (edge.bpmnElement instanceof MessageFlow && edge.messageVisibleKind !== MessageVisibleKind.NONE) {
+      const mxCell = this.graph.insertVertex(mxEdge, `message_${mxEdge.id}`, undefined, 0, 0, 20, 14, this.styleConfigurator.computeMessageStyle(edge));
```

Comment:
:question: can we use something that provides more context instead of only `message`?